src/node_process.cc

```diff
@@ -434,6 +434,30 @@ void SetEUid(const FunctionCallbackInfo<Value>& args) {
 }
 
 
+void Nice(const FunctionCallbackInfo<Value>& args) {
+  Environment* env = Environment::GetCurrent(args);
+  CHECK(env->is_main_thread());
+
+  CHECK_EQ(args.Length(), 1);
+
+  int inc = 0;
+  // ..only check type if argument is int32
+  if (!args[0]->IsUndefined()) {
+    CHECK(args[0]->IsInt32());
+    inc = args[0]->Int32Value();
```

Comment:
Can you use `args[0].As<Int32>()->Value()`? `->Int32Value()` is an operation that potentially performs a type conversion, but that’s unnecessarily complex, since we already know that it’s an Int32

Reply:
Sounds great and it's done, thanks!